checks/rolebindingClusterRolePodExecAttach.yaml

```diff
@@ -0,0 +1,94 @@
+successMessage: The RoleBinding does not reference a ClusterRole allowing pods/exec or pods/attach
```

Comment:
messaging still refers to `ClusterRole` in role binding

Reply:
I believe this message is correct, because the primary schema looks at bindings, and the additional schema looks at the ClusterRole associated with the binding.